meson.build

```diff
@@ -107,6 +107,7 @@ rc_bindir = rc_libexecdir / 'bin'
 rc_sbindir = rc_libexecdir / 'sbin'
 rc_shdir = rc_libexecdir / 'sh'
 sbindir = rootprefix / get_option('sbindir')
+pamdir =	get_option('sysconfdir') / 'pam.d'
```

Comment:
we don't use tabs for indentation elsewhere in here

also, this doesn't seem like it's related to this change (moving programs to dedicated dirs).  seems like this is a bugfix in general that should be in a sep commit for fixing pam.d install ?